client/dashboard/sites/overview/site-card.tsx

```diff
@@ -8,25 +9,28 @@ import {
 } from '@wordpress/components';
 import { dateI18n } from '@wordpress/date';
 import { __, sprintf } from '@wordpress/i18n';
+import { sitePHPVersionQuery } from '../../app/queries';
 import { getSiteStatusLabel } from '../../utils/site-status';
 import SitePreview from '../site-preview';
 import type { Site, SiteDomain, Plan } from '../../data/types';
 
+function PHPVersion( { siteSlug }: { siteSlug: string } ) {
+	return useQuery( sitePHPVersionQuery( siteSlug ) ).data;
+}
+
 /**
  * SiteCard component to display site information in a card format
  */
 export default function SiteCard( {
 	site,
-	phpVersion,
 	primaryDomain,
 	currentPlan,
 }: {
 	site: Site;
-	phpVersion?: string;
 	primaryDomain?: SiteDomain;
 	currentPlan: Plan;
 } ) {
-	const { options, URL: url, is_private } = site;
+	const { options, URL: url, is_private, is_wpcom_atomic } = site;
```

Comment:
I'm just curious, why not just call the original

```tsx
const { data: phpVersion } = useQuery( {
		...sitePHPVersionQuery( siteSlug ),
		enabled: site?.is_wpcom_atomic,
	} );
```

here and then render `phpVersion` as before. Is there any particular reason of using a `PHPVersion` React component? Is it because we don't want to pass the `{ enabled: ...}` option?

Reply:
Btw, while working on another PR this was still returning data for me even though I disabled it. It seems `enabled` only stops it from being reactive, I'm not sure 🤔

Reply:
Interesting... I'll also try to experiment with it 😱